src/gobapi/core/model/__init__.py

```diff
@@ -0,0 +1,68 @@
+from gobcore.model import GOBModel
+
+
+def get_catalogs():
+    """Catalogs
+
+    Returns a list of catalog names
+
+    :return:
+    """
+
+    # todo: this should be read from configuration file(s)
+
+    return {
+        'meetbouten': {
+            'description': 'De meetgegevens van bouten in de Amsterdamse panden',
+            'collections': GOBModel().get_model_names()
+        }
+    }
+
+
+def get_catalog_names():
+    """Catalog Names
+
+    Returns the names in the list of catalogs
+
+    :return:
+    """
+    return get_catalogs().keys()
+
+
+def get_catalog(catalog_name):
+    """Catalog
+
+    Returns the catalog in the list of catalogs with the specified name
+    If the name does not exist, None is returned
+
+    :param catalog_name:
+    :return:
+    """
+    catalogs = get_catalogs()
+    return get_catalogs()[catalog_name] if catalog_name in catalogs else None
```

Comment:
Couldn't this just be:

`catalogs[catalog_name] if catalog_name in catalogs else None`

Reply:
It not only could be that, it should be that :-)